src/toplevel.c

```diff
@@ -616,6 +616,12 @@ jl_value_t *jl_toplevel_eval_flex(jl_module_t *JL_NONNULL m, jl_value_t *e, int
             while (*n == '_') ++n;
             if (*n == 0 && n > n0)
                 jl_eval_errorf(m, "all-underscore identifier used as rvalue");
+            // .op needs special lowering
+            if (jl_is_dotop_named(n)) {
```

Comment:
Should this code be here or in `eval_value`?

Reply:
It seemed similar enough to the r-value check, so I put it here. I can certainly change that, though.

Reply:
I guess this place is good — the rvalue check is here because the symbol didn't hit the lowering code (which also checks for underscore names), and so we want the dotop check here for the same reason.